packages/react/src/components/header/components/headerActionBarItem/HeaderActionBarItem.module.scss

```diff
@@ -1,87 +1,156 @@
-@use '../../../../styles/common.scss';
-
-button.actionBarItem {
-  align-self: center;
-  appearance: none;
-  background: transparent;
-  border: none;
-  color: var(--header-color);
-  cursor: pointer;
-  padding: 0;
-
-  &:focus-visible {
-    border: none;
-    outline: var(--header-focus-outline-width) solid var(--header-focus-outline-color);
-    outline-offset: var(--header-focus-outline-width);
-  }
+.dropdownWrapper {
+  bottom: 0;
+  overflow: hidden;
+  position: absolute;
+  right: 0;
+  transform: translateY(100%) translateY(1px);
+  transition-duration: var(--animation-duration-dropwdown);
+  transition-property: max-height, padding-bottom;
+  transition-timing-function: ease;
+  z-index: 20;
 }
 
-.actionBarItemIcon {
-  display: block;
-  height: var(--icon-size);
-  margin: 0 auto;
-  padding: 3px 0 0;
-  pointer-events: none;
-  width: var(--icon-size);
+.container {
+  display: flex;
 
-  & > svg {
-    display: block;
-    height: 100%;
-    width: auto;
+  .dropdownWrapper {
+    max-height: 0;
+    min-width: 320px;
+    padding-bottom: 0;
+    transition-duration: var(--animation-duration-dropwdown);
+    transition-timing-function: ease;
   }
-}
 
-.actionBarItemLabel {
-  color: var(--action-bar-item-title-font-color);
-  display: block;
-  font-size: var(--action-bar-item-title-font-size);
-  margin: 0 auto;
-  pointer-events: none;
-  white-space: nowrap;
+  &.visible .dropdownWrapper {
+    max-height: 1000vh;
+    padding-bottom: 20px;
```

Comment:
Is there a `--spacing ` variable for this 20px?

Reply:
fixed